### PR TITLE
LPD-33257 Load fields also in mapping selector

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:ejecutalostestdeplaywrightdepagemanagement

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/MappingSelector.js [hunk 1]
```diff
@@ -174,6 +175,33 @@ function getInitialSourceType(mappedItem, relationship) {
 	return MAPPING_SOURCE_TYPES.content;
 }
 
+const loadCollectionFields = (
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/MappingSelector.js:178]: We have the same code in the `StructureTreeNode` [here](https://github.com/liferay/liferay-portal/blob/13f08313712483f1d8f37ed3083d849ccaf37ed4/modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTreeNode.js#L79), can we unify it?

**Comment by veroglez** [comment]
Hey @victorg1991! it works perfectly. I just left a small comment :smile:

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/MappingSelector.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/MappingSelector.js]: I usually go by the [rule of three](https://wou.edu/las/cs/csclasses/cs161/Lectures/rulesofthumb.html#) but I changed it nonetheless. Thanks vero! :)

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**